utils/semantics.py

```diff
@@ -0,0 +1,241 @@
+# declaring them as global variables so that we can change the type to fit (string, int, enum) later
+INT = 'int'
+FLOAT = 'float'
+CHAR = 'char'
+BOOL = 'bool'
+ERR = 'err'
+plus = '+'
+minus = '-'
+div = '/'
+mult = '*'
+andop = '&&'
+orop = '||'
+morethan = '>'
+lessthan = '<'
+moreequal = '>='
```

Comment:
I see `<=` and `>=` here but they're not declared in our lexicon. We should add them there too, right?